src/ray/protobuf/serve.proto

```diff
@@ -60,6 +60,9 @@ message AutoscalingConfig {
 
   // The multiplicative "gain" factor to limit downscale.
   optional double downscale_smoothing_factor = 11;
+
+  // The cloudpickle base64 serialized callable for the autoscaling policy.
```

Comment:
Based on [this comment](https://github.com/ray-project/ray/pull/42072/files#diff-2f5ba63ace082b7b7cfeceb2baf9b2d9f805f7559ac81e1fa5878e7b33aa69b9R98-R101), it sounds like this can also be an import path. If so, could you mention that in this docstring? If not, could you change that comment?

```suggestion
  // The cloudpickled base64-serialized callable for the autoscaling policy.
```